### ISSUE TITLE
Ads not appearing + ad blocker detected on a website with uBO disabled

### ISSUE THREAD

**Issue by Alex4567890** [uBlockOrigin/uBlock-issues#1490]
### Prerequisites

- [X] I verified that this is not a filter issue
    - Filter issues MUST be reported at [filter issue tracker](https://github.com/uBlockOrigin/uAssets/issues)
    
- [X] This is not a support issue or a question
    - Support issues and questions are handled at [/r/uBlockOrigin](https://old.reddit.com/r/uBlockOrigin/)
    
- [X] I performed a cursory search of the issue tracker to avoid opening a duplicate issue
    - Your issue may already be reported.
- I tried to reproduce the issue when...
    - [X] uBlock Origin is the only extension
    - [X] uBlock Origin with default lists/settings
    - [X] using a new, unmodified browser profile
- [X] I am running the latest version of uBlock Origin
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I report is not a normal behavior

### Description

So, ads are being blocked even if uBO is disabled.

### A specific URL where the issue occurs

[A specific URL is MANDATORY for issue happening on a web page, even if it happens "everywhere"]

### Steps to Reproduce

* I am not sure if I can reproduce it.

### Expected behavior:

[What you expected to happen]

### Actual behavior:

Basically, I reported a message about my ad blocker on a website and I found that the only way to remove the message was to remove it / hide it / prevent it or whatever from appearing using the element picker. I'm not sure what happened, but maybe I closed Google Chrome before closing uBO's logger.
Actually, it probably has nothing to do with the website I was trying to get fixed or whatever.

### Your environment

* uBlock Origin version: 1.33.2
* Browser Name and version: Google Chrome Version 88.0.4324.146 (Official Build) (64-bit)
* Operating System and version: Microsoft Windows 10 Pro Version 10.0.19042 Build 19042

**Comment by Alex4567890** [comment]
Nevermind lol, Adblock Plus was turned on and not pinned, I thought I desabled it or whatever.
